### PR TITLE
fix: Validate if record holder is in contributors list

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[BUG] Dirty angular sessions on the UI](https://trello.com/c/Vc2t9Ull/43-bug-dirty-angular-sessions-on-the-ui)

**Comment by DanielPalafox** [comment]
[[BUG] Wrong contributor ORCID on records](https://trello.com/c/xS1NSfNe/42-bug-wrong-contributor-orcid-on-records)

**Comment by DanielPalafox** [comment]
[[BUG] Undefined contributors](https://trello.com/c/aTzqTvTd/41-bug-undefined-contributors)

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java [hunk 1]
```diff
@@ -543,6 +545,11 @@ private void initializePublicationDate(WorkForm w) {
         validateWork(workForm);
         if (workForm.getErrors().size() == 0) {
             removeEmptyExternalIdentifiers(workForm);
+            if (isRecordHolderInContributors(workForm)) {
```
Comment (amontenegro) [inline orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java:548]: @DanielPalafox here you are missing a `!` right? it should be "if record holder is NOT in contributors"

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java (hunk 1 above)
Comment (DanielPalafox) [reply orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java]: done! I rename the method

**Review by amontenegro — APPROVED** [review]

**Pull request merged**